blog/_posts/2016-07-15-team-joda-says-hello.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Team joda says hello
+layout: post
+created_at: Thu Jul 14 2016
+permalink: blog/2016-07-14-team-joda-says-hello
```

Comment:
permalink should always be the same date as the filename. If we want to publish today, can you please change the filename to `2016-07-14-team-joda-says-hello.md`? thank you :)